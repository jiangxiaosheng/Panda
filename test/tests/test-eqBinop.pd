func main() {
	var x = 1
    var y = 2

    x == y
    x != y
    x > y
    x < y
    x >= y
    x <= y

    var t = true
    var f = false

    t || f
    t && f
}