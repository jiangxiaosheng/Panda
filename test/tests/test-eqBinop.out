

Sementically checked program: 


void main() {
var x: int(int : 1)
var y: int(int : 2)
(bool : (int : x) == (int : y))
(bool : (int : x) != (int : y))
(bool : (int : x) > (int : y))
(bool : (int : x) < (int : y))
(bool : (int : x) >= (int : y))
(bool : (int : x) <= (int : y))
var t: bool(bool : true)
var f: bool(bool : false)
(bool : (bool : t) || (bool : f))
(bool : (bool : t) && (bool : f))
}

